### PR TITLE
lesson 3

### PR DESCRIPTION
**Pull request by Dilstom**

### CODE REVIEW

File: lessons/find_number.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
+
+def find_number
+  secret_number = rand(1..100)
+  
+  loop do
+    puts "Guess a number from 1 to 100"
+    guessed_number = gets.chomp.to_i
+      
+    puts guessed_number
+    puts secret_number
```
Comment (dh1011) [inline lessons/find_number.rb]: comment out or remove

File: lessons/second.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+puts "What is your age?"
+people[:age] = gets.chomp
+puts "What is your  street address?"
+people[:street_address] = gets.chomp
+puts "What is your city?"
+people[:city] = gets.chomp
+puts "What is your state?"
+people[:state] = gets.chomp
+
+# people = {:first_name=>"sim", :last_name=>"si", :age=>"211", :street_address=>"ret", :city=>"gr", :state=>"ca"}
```
Comment (dh1011) [inline lessons/second.rb]: remove

File: lessons/second.rb [hunk 3]
```diff
@@ -0,0 +1,28 @@
+
+people = {}
+
+puts "What is your first name?"
+people[:first_name] = gets.chomp
+puts "What is your last name?"
+people[:last_name] = gets.chomp
+puts "What is your age?"
+people[:age] = gets.chomp
```
Comment (dh1011) [inline lessons/second.rb]: not required by the assignment but consider a check for numeric input

File: lessons/guesser.rb [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+   
+  loop do
+    if min_num > max_num || steps >= 7
+      puts "Oops, you seem to be lying as I exhausted the possible answers"
+      break
+    end
+
+    mid_num = min_num + (max_num - min_num)/2
+
+    puts "Is it #{mid_num}? Y/y - correct, Too loo - L/l, Too high - H/h";
```
Comment (dh1011) [inline lessons/guesser.rb]: typo, 'Too low'

File: lessons/find_number.rb (hunk 1 above)
Comment (Dilstom) [reply lessons/find_number.rb]: done

File: lessons/second.rb (hunk 2 above)
Comment (Dilstom) [reply lessons/second.rb]: done

File: lessons/second.rb (hunk 3 above)
Comment (Dilstom) [reply lessons/second.rb]: done

File: lessons/guesser.rb (hunk 4 above)
Comment (Dilstom) [reply lessons/guesser.rb]: done )

**Pull request merged**